ipv4_address.go

```diff
@@ -20,28 +20,48 @@ package hedera
  *
  */
 
-type _IPv4Address struct {
-	network _IPv4AddressPart
-	host    _IPv4AddressPart
+type IPv4Address struct {
```

Comment:
`IPv4` addresses would typically be represented as four octets (bytes), no two parts (network and host). While it's possible to divide an `IPv4` address into network and host parts, the representation here is non-standard. Is there a specific reason it is implemented like this?

Reply:
This has been the design since the creating of the SDK, I'm not really sure.

Reply:
Maybe it's the perfect time to fix this 🤣 But it is not important if it works already.

Reply:
Could do some testing in a separate task.